src/components/Movies.js

```diff
@@ -0,0 +1,59 @@
+import "../App.css";
+import React from "react";
+
+
+class Movies extends React.Component {
+  constructor(){
+    super()
+
+    this.state ={
+      films:[],
+      currentFilms: null,
+    };
+  }
+
+  fetchFilms = () =>{
+    fetch("https://ghibliapi.herokuapp.com/films")
+    .then((res) => res.json())
+    .then((data)=> {
+      this.setState({
+        films: data,
+      });
+    }) ;
+  };
+  
+  componentDidMount = () =>{
+    this.fetchFilms();
+  }
+
+
+  handleDropDown = (event) =>{
+    let currentMovieObj = this.state.films.find((film) =>{
+      return film.title === event.target.value
+    })
+  }
+
+    render(){
+
+      let dropDownOptions = this.state.films.map((film) =>{
+        return <option>{film.title}</option>;
+      })
+
+
+
+  return (
+    <div className="movies">
+      
+        <h1>Select a Movie</h1>
+        <select onChange={this.handleDropDownChange}>
+          <option>Select a Movie</option>
+          {dropDownOptions}
+
+          </select>
+      
```

Comment:
```suggestion
       {this.state.selectedMovie ? (
          <div className="movie-info">
            <h2>{this.state.currentFilm.title}</h2>
            <p>{this.state.currentFilm.release_date}</p>
            <p>{this.state.currentFilm.description}</p>
          </div>
        ) : null}
```
this is where you would use `this.state.currentFilm` to display the appropriate movie info!